honeybee_vtk/tovtk.py

```diff
@@ -0,0 +1,311 @@
+"""Functions to create vtk objects based on the data extracted from hbjson."""
+
+import vtk
+
+
+def group_by_face_type(points_lst, hb_types):
+    """Group points based on Honeybee type.
+
+    Args:
+        points_lst: A list of lists of points for the geometry in the HBJSON. Here,
+            each point is a list of X, Y, and Z cordinates of the point.
+        hb_types: A list of honeybee face_types and/or types for the geometry objects
+            in the HBJSON.
+
+    Returns:
+        A dictionary with Honeybee type as keys and list of list of points for geometry
+        that belongs to that Honeybee type.
+    """
+    grouped_points = {face_type: [] for face_type in hb_types}
+
+    for i in range(len(hb_types)):
+        grouped_points[hb_types[i]].append(points_lst[i])
```

Comment:
Can you provide a small sample here. I'm very confused on how this is supposed to work. Probably a test can also be very helpful.

Reply:
Example provided